packages/react/src/components/CodeSnippet/CodeSnippet.mdx

```diff
@@ -0,0 +1,54 @@
+import { Story, Props, Source, Preview } from '@storybook/addon-docs/blocks'
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs'
+import * as stories from './CodeSnippet-story.js'
+import CodeSnippet from '../CodeSnippet'
+
+<!-- How do I get <Props> to work?
+ -->
+# CodeSnippet
```

Comment:
Should include a table of contents at the top (can refer to accordion for structure)